helm_deploy/help-with-prison-visits-asynchronous-worker/templates/worker-tasks-cron.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-cron-worker-tasks
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "* * * * *"
```

Comment:
this is every minute?  How long does the job take to run?

Reply:
It was actually set for every 5 seconds under the JS cron stuff as it allowed that.